target/linux/ipq40xx/files-4.14/arch/arm/boot/dts/qcom-ipq4019-qxwlan-e2600ac.dtsi

```diff
@@ -0,0 +1,238 @@
+/* SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+ *
+ * Copyright (c) 2018 Peng Zhang <sd20@qxwlan.com>
+ *
+ */
+
+#include "qcom-ipq4019.dtsi"
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/soc/qcom,tcsr.h>
+
+/ {
+
+	model = "Qxwlan E2600AC";
+	compatible = "qxwlan,e2600ac", "qcom,ipq4019";
```

Comment:
Is there a "qxwlan,e2600ac" device or series? If not you could just delete the compatible line here.